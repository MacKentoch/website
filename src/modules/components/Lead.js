import styled from 'styled-components'
import theme from 'style/theme'

const Lead = styled.p`
  font-size: 28px;
  font-weight: 300;
  line-height: 1.2;
  margin: 0;
  letter-spacing: 0.05em;

  @media (min-width: ${theme.medias.phablet}) {
    font-size: 30px;
  }
`

export default Lead
