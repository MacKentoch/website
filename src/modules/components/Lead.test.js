import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Lead from './Lead'

describe('Lead', () => {
  it('should render a p', () => {
    const wrapper = shallow(<Lead>Hello</Lead>)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
