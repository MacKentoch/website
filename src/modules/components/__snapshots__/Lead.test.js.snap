// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lead should render a p 1`] = `
<p
  className="Lead__Lead-s78qw09-0 cytdxL"
>
  Hello
</p>
`;
